http/src/request/application/delete_followup_message.rs

```diff
@@ -0,0 +1,91 @@
+use crate::{
+    client::Client,
+    error::Result,
+    request::{Pending, Request},
+    routing::Route,
+};
+use twilight_model::id::{ApplicationId, MessageId};
+
+/// Delete a followup message created from a interaction.
+///
+/// # Examples
+///
+/// ```no_run
+/// # use twilight_http::Client;
+/// use twilight_model::id::{MessageId, ApplicationId};
+///
+/// # #[tokio::main]
+/// # async fn main() -> Result<(), Box<dyn std::error::Error>> {
+/// # let client = Client::new("token");
```

Comment:
We don't really need to hide the instantiation of the client, other examples use this I believe:

```suggestion
/// # #[tokio::main] async fn main() -> Result<(), Box<dyn std::error::Error>> {
/// use std::env;
/// use twilight_http::Client;
/// use twilight_model::id::{ApplicationId, MessageId};
///
/// let client = Client::new(env::var("DISCORD_TOKEN")?);
```

This is true for other examples in this PR.